### PR TITLE
CNV 253 HCO bump and upgrade doc

### PR DESCRIPTION
**Pull request by aburdenthehand**
Bumping the HCO version for z-stream, and massaging the upgrade pathways doc.

### CODE REVIEW

File: modules/virt-upgrade-pathways.adoc [hunk 1]
```diff
@@ -13,13 +13,15 @@ You must upgrade your {product-title} to 4.7 before upgrading the minor release
...
+== Upgrading from 2.4.3 to {HCOVersion}
 
-You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2.
+You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2 and up. 
 
 If your *Approval Strategy* is *Automatic*, which is the default, {VirtProductName} upgrades the z-stream automatically once you have upgraded to 2.5.0.
 
 [id="virt-upgrade-pathways-2.4.4_{context}"]
-== Upgrading from 2.4.4 to 2.5.2
+== Upgrading from 2.4.4 to {HCOVersion}
```
Comment (tiraboschi) [inline modules/virt-upgrade-pathways.adoc:23]: Now it should be "Upgrading from 2.4.4/2.4.5 to {HCOVersion}"

Comment (aburdenthehand) [reply modules/virt-upgrade-pathways.adoc]: Updated here and subsequent line to "2.4.4 or 2.4.5"

File: modules/virt-upgrade-pathways.adoc [hunk 2]
```diff
@@ -13,13 +13,15 @@ You must upgrade your {product-title} to 4.7 before upgrading the minor release
 ====
 
 [id="virt-upgrade-pathways-2.4.3_{context}"]
-== Upgrading from 2.4.3 to 2.5.2
+== Upgrading from 2.4.3 to {HCOVersion}
 
-You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2.
+You must first upgrade to 2.5.0 before you can upgrade the z-stream. You can then upgrade from 2.5.0 to 2.5.1 and then to 2.5.2 and up.
```
Comment (ousleyp) [inline modules/virt-upgrade-pathways.adoc:18]: "and up" is giving me pause, but I don't have a strong argument as to why.

File: modules/virt-upgrade-pathways.adoc [hunk 3]
```diff
@@ -13,13 +13,15 @@ You must upgrade your {product-title} to 4.7 before upgrading the minor release
...
 If your *Approval Strategy* is *Automatic*, which is the default, {VirtProductName} upgrades the z-stream automatically once you have upgraded to 2.5.0.
 
 [id="virt-upgrade-pathways-2.4.4_{context}"]
-== Upgrading from 2.4.4 to 2.5.2
+== Upgrading from 2.4.4 or 2.4.5 to {HCOVersion}
 
-You can upgrade from 2.4.4 directly to 2.5.2.
+You can upgrade from 2.4.4 or 2.4.5 directly to 2.5.2. You can then upgrade from 2.5.2 to 2.5.3.
+
+If your *Approval Strategy* is *Automatic*, which is the default, {VirtProductName} upgrades the z-stream automatically once you have upgraded to 2.5.2.
```
Comment (ousleyp) [inline modules/virt-upgrade-pathways.adoc:27]: ISG p350
! once adv
Use only to mean “one time.” Do not use as a conjunction to mean “after” or “when.”

**Review by ousleyp — APPROVED** [review]
Overall LGTM, just a nit (and a half)

File: modules/virt-upgrade-pathways.adoc (hunk 3 above)
Comment (aburdenthehand) [reply modules/virt-upgrade-pathways.adoc:27]: Got me good

File: modules/virt-upgrade-pathways.adoc (hunk 2 above)
Comment (aburdenthehand) [reply modules/virt-upgrade-pathways.adoc:18]: Yeah, I know what you mean. Versions should be talked about as 'earlier' and 'later' which doesn't really work here either.
But turns out 'and so on', which is what I was avoiding, is blessed by the ISG.

**Pull request merged**